development/src/utility/hexagons.js

```diff
@@ -666,6 +645,36 @@ var HexGrid = Class.create( {
 		});
 	},
 
+	/* getHexLine(x, y, dir, flipped)
```

Comment:
Tested. I think this change broke all unit abilities. You should had probably used the initial code abilities.js within Swine Thug.js file without actually copy-pasting it and it should have worked fine, as most abilities were already using that piece of code for targeting and such.